### DISCUSSION TITLE
Lunar Cartographic Standards | Planetary SDI

### DISCUSSION THREAD

**Discussion by giscus** [DOI-USGS/planetary-sdi#9] [General]
# Lunar Cartographic Standards | Planetary SDI

DRAFT
Map Projections Data are provided in a simple cylindrical (equirectangular) map projection. Data are orthorectified to a sphere using the IAU radius => 1737.4km (Archinal, et al. 2018) Data are made available in 0 – 360 positive East coordinate system with a center longitude of 180 degrees using planetocentric latitude. For observational data, pixel scale or resolution should be maintained at native scale or better, (i.e., do not average data.

https://psdi.astrogeology.usgs.gov/moon/standards/cartographic_standards/

**Comment by mkbarker** [comment]
"... that avoids interpolation of information." seems a bit overly restrictive. Interpolation is sometimes necessary.
